config/default/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+# Adds namespace to all resources.
+namespace: default
+
+namePrefix: ibm-block-csi-operator-
```

Comment:
does [this](https://github.com/kubernetes-sigs/kubebuilder/blob/v3.1.0/docs/book/src/component-config-tutorial/testdata/project/config/default/kustomization.yaml#L4-L6) mean that e.g. our cluster role will now be named `ibm-block-csi-operator-ibm-block-csi-operator`?

Reply:
it seems that it doesn't impose the prefix
I can remove this line

Reply:
I think every "dead" lines should be removed (this is just an example)